src/services/ui/src/pages/form/medicaid-form.tsx

```diff
@@ -0,0 +1,349 @@
+            <p>
+              Once you submit this form, a confirmation email is sent to you and
+              to CMS. CMS will use this content to review your package, and you
+              will not be able to edit this form. If CMS needs any additional
+              information, they will follow up by email.{" "}
+              <strong className="bold">
+                If you leave this page, you will lose your progress on this
+                form.
+              </strong>
+            </p>
+          </section>
+
+          <I.FormField
+            control={form.control}
+            name="id"
+            render={({ field }) => (
+              <I.FormItem>
+                <div className="flex justify-between">
+                  <I.FormLabel>SPA ID</I.FormLabel>
+                  <Link to="/faq" className="text-blue-700 hover:underline">
+                    What is my SPA ID?
+                  </Link>
+                </div>
+                <p>Must follow the format SS-YY-NNNN or SS-YY-NNNN-xxxx.</p>
+                <p className="italic">
+                  Reminder - CMS recommends that all SPA numbers start with the
+                  year in which the package is submitted.
+                </p>
+                <I.FormControl className="max-w-sm">
+                  <I.Input {...field} />
+                </I.FormControl>
+                <I.FormMessage />
+              </I.FormItem>
+            )}
+          />
+
+          <I.FormField
+            control={form.control}
+            name="proposedEffectiveDate"
+            render={({ field }) => (
+              <I.FormItem className="max-w-sm">
+                <I.FormLabel className="block">
+                  Proposed Effective Date of Medicaid SPA
+                </I.FormLabel>
+                <I.FormDescription>For example: 4/28/1986</I.FormDescription>
+                <I.FormControl>
+                  <I.DatePicker onChange={field.onChange} date={field.value} />
+                </I.FormControl>
+                <I.FormMessage />
+              </I.FormItem>
+            )}
+          />
+
+          <section>
+            <h3 className="text-2xl font-bold font-sans">Attachments</h3>
+            <p>
+              Maximum file size of 80 MB per attachment. You can add multiple
+              files per attachment type.Read the description for each of the
+              attachment types on the FAQ Page. We accept the following file
+              formats:{" "}
+              <strong className="bold">
+                .doc, .docx, .jpg, .odp, .ods, .odt, .png, .pdf, .ppt, .pptx,
+                .rtf, .txt, .xls, .xlsx,
+              </strong>{" "}
+              and a few others. See the full list on the FAQ Page. *At least one
+              attachment is required.
+            </p>
+          </section>
+
+          {attachmentList.map((attachment) => (
+            <I.FormField
+              key={attachment[0]}
```

Comment:
Can we de-structure `attachment` to make this make more sense? You might be able to read it in as `[key, label]` on line 283, but you can also do `const [key, label] = attachment` within a function block`